base/docs/basedocs.jl

```diff
@@ -1841,32 +1888,67 @@ See also: [`digits`](@ref), [`iszero`](@ref), [`NaN`](@ref).
 """
 Bool
 
-for (bit, sign, exp, frac) in ((16, 1, 5, 10), (32, 1, 8, 23), (64, 1, 11, 52))
-    @eval begin
-        """
-            Float$($bit) <: AbstractFloat
+"""
+    Float64 <: AbstractFloat <: Real
```

Comment:
The reason to add `<: Real` is seeing one too many functions typed `f(x::AbstractFloat)`.